airflow/hooks/hive_hooks.py

```diff
@@ -765,7 +784,14 @@ def get_conn(self, schema=None):
             user=db.login,
             database=schema or db.schema or 'default')
 
-    def get_results(self, hql, schema='default', arraysize=1000):
+    def get_results(self, hql, schema='default', hive_conf=None):
```

Comment:
This changes the Hive API, please note this in `UPDATING.md`

Reply:
Will do that. Can't help wondering, does it mean we're supporting usage of hooks for users? I was under the impression that hooks are more for airflow internal use.